docs/hooli-demo-idv.mdx

```diff
@@ -15,14 +15,4 @@ Additional information on how to issue credentials can be found in Free IDV [gui
 ## Web Demo
 The live Hooli web application demo can be found [here](https://hooli-web.demo.sandbox-unumid.co/login). Its open source code can be found on [Github](https://github.com/UnumID/Hooli-Demo-Web).
 
-## Mobile Demos
-The mobile demo showcase how to achieve the same functionality as the web demo in a native app.
-
-### Android
-The Hooli Android Demo can be installed from [here](https://appdistribution.firebase.dev/i/f59869e16365c7f8). The source code can be found on [Github](https://github.com/UnumID/UnumIDWebWallet-android).
-
-### iOS
-The Hooli iOS Demo can be installed from [here](https://testflight.apple.com/join/kjdN0EAD). The Hooli iOS Demo source code can be found on [Github](https://github.com/UnumID/Hooli-iOS).
-
-## Preview
-![](/img/hooli-ios.png)
+Feel free to use a real email address that you control or our test account, richard@piedpiper.net. Whatever value used, the password field on the demo login page is irrelevant for the purposes of the demo. It is important use the same value if then continuing to the [Kredita](/kredita-demo) demo. If opting to use Richard test account, you can use the static OTP of `111111` to authenticate to the [wallet](https://wallet-beta.sandbox-unumid.co/cards) - a necessary step in the Kredita demo.
```

Comment:
"It is important use the same value" — this sounds like same value for the password. I'd just change "value" to "email" here. (Side note: can I change small things for you in this review process, to make it easier for you?)

Reply:
Yes definitely please make the changes. GH actually provides an easy way to make "suggestions" which can be committed in a click. https://verified-inc.slack.com/archives/C01DJG1EPKP/p1677607582245319